src/main/java/appeng/fluids/util/AEFluidTank.java

```diff
@@ -16,30 +16,64 @@
  * along with Applied Energistics 2.  If not, see <http://www.gnu.org/licenses/lgpl>.
  */
 
-package appeng.fluids.helper;
+package appeng.fluids.util;
 
 
+import net.minecraft.tileentity.TileEntity;
 import net.minecraftforge.fluids.FluidTank;
 
+import appeng.api.storage.data.IAEFluidStack;
+import appeng.util.Platform;
 
-public class AEFluidTank extends FluidTank
+
+public class AEFluidTank extends FluidTank implements IAEFluidTank
 {
 	private final IAEFluidInventory host;
 
 	public AEFluidTank( IAEFluidInventory host, int capacity )
 	{
 		super( capacity );
 		this.host = host;
+		if (host instanceof TileEntity)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)